tomcat-embedded-6/pom.xml

```diff
@@ -20,6 +20,20 @@
 
   <dependencies>
 
+    <dependency>
+      <groupId>org.jboss.arquillian.container</groupId>
+      <artifactId>arquillian-tomcat-ee8-deps</artifactId>
```

Comment:
Why do we need it?

Reply:
Good catch, we should only need this dep with `<type>test-jar</type>`. I'll adjust this in all affected modules.